lib/util/renderMarkdown.ts

```diff
@@ -0,0 +1,7 @@
+import { MarkdownPreviewView } from "obsidian";
+
+export const renderMarkdown = (markdown: string) => {
+  const el = document.createElement("span");
+  MarkdownPreviewView.renderMarkdown(markdown, el, "", null);
```

Comment:
I get the following console error when loading the plugin:

![image](https://github.com/mcndt/obsidian-toggl-integration/assets/23149353/2a9e07d1-3e50-4753-871c-054aee3542b0)

Probably related to the `null` argument here.

Reply:
Good catch! I'll have to find a way to pass the reference through the context, I think.